omero/developers/Web/Deployment.txt

```diff
@@ -58,142 +58,7 @@ Debugging on and start the server up:
 Using FastCGI (Unix/Linux)
 --------------------------
 
-Once you have installed a FastCGI capable web server, configuration of
```

Comment:
The content here is useful for deployment, not really related to development.  A sysadmin reading through the install docs is not going to look at something hidden in the developer docs.  I think this change is an improvement in the organisation of the docs.